### PR TITLE
Update to Babel 6

### PR DESCRIPTION
**Pull request by sapegin**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
-  "comments": false,
-  "loose": "all"
+  "presets": [
+    ["env", {
```
Comment (developit) [inline .babelrc:3]: does loose fall to es2015 automatically? I thought it had to be nested under `{ es2015: { loose: true } }`

Comment (sapegin) [reply .babelrc:3]: From env preset docs:

> Enable "loose" transformations for any plugins in this preset that allow them.

**Review by developit — APPROVED** [review]

**Comment by developit** [comment]
Looks like this enabled the `Symbol` typeof fix, which we'll need to find a way to turn off.

**Comment by sapegin** [comment]
You mean this line?

```js
 var _typeof6 = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
```

**Comment by developit** [comment]
yup yup. repeats a bunch of times - the original code is checking for `typeof Symbol!=='undefined'`, but that triggers Babel to polyfill typeof.  I can't remember the way to avoid that in 6.x

**Comment by sapegin** [comment]
Looks like that’s it: [transform-es2015-typeof-symbol](https://babeljs.io/docs/plugins/transform-es2015-typeof-symbol/) so we can try to disable it in `env` preset settings.

**Comment by developit** [comment]
Yup. Not actually sure how to disable plugins at this point, I used to use `modify-babel-preset` but it's sortof broken now

**Comment by sapegin** [comment]
Something like this I think — https://github.com/babel/babel-preset-env#exclude

```json
{
  "presets": [
    ["env", {
      "exclude": ["transform-es2015-typeof-symbol"]
    }]
  ]
}
```

**Comment by developit** [comment]
ahhhhh that's amazing

**Pull request merged**